4.3 Update

JulanDeAlb opened the pull request.

JulanDeAlb committed:
4.3 Update

noproto commented on application.fam:
I think we need to target f7? When I've PR'd apps before the developers always add that target.

JulanDeAlb replied:
Ok i have found this: [.fap Docs](https://github.com/flipperdevices/flipperzero-firmware/blob/f9101d80840b4b7ef7146e41bba99aac881bbfca/documentation/AppManifests.md?plain=1#L43)
When using ufbt it doesnt matter if i set the target like this or not, it will only build for target f7. Maybe its more important for potential upcoming devices.
But i can still add it back.

JulanDeAlb committed:
Removal of .flipcorg

JulanDeAlb committed:
Update to 6.6

JulanDeAlb committed:
Updated floods

JulanDeAlb committed:
Create build.yml

noproto commented:
Yeah sorry about this, please use https://github.com/wrenchathome/ofw_ble_spam now.

JulanDeAlb commented:
Ah yeah was just a mistake, renamed my remote branch but commited to master again. 
I've updated to the latest 6.6 (just had to remove one feature and some minor stuff because the ofw api doesnt have it)

Edit: Ah i see, he got something that i removed, i will fork it then and add a workflow for easier access to the faps.
